docs/styling.md

```diff
@@ -45,3 +45,53 @@ MyButton.register();
 !> __Important ⚠️:__ Style-Scoping only works in browsers that support shadow dom. Browsers like IE11/Edge V18 and older inherit styles as used everywhere else. So use unique class names and techniques like BEM to keep everything sane.
 
 !> __Important ⚠️:__ If you come across that issue, that your styles are displayed as text. Most likely you use a class-name that is the same as your component name. Together with the polyfill, the style tag itself get styled which causes the issue in IE11/Edge V18 and below.
+
+## Dynamic styles
+If your styles are dynamic and depending on some values in the javascript context, just use inline styles in your template string:
+
+```js
+import Component from '@biotope/element';
+
+class MyButton extends Component {
+  color: 'orange';
+  render() {
+    return this.html`
+      <style>
```

Comment:
CSS values set with js is an anti pattern and should only be used as some kind of last resort, do we really want to have that in our docs? If yes i vote for adding a reminder that you should only use inline styles and js generated css styles when ther eis no other possible solution.

Reply:
Can you further explain? Setting style in shadow dom always requires JS. Setting dynamic styles could be either this or using custom css properties which is not supported widely enough, so this is the proposed way :)

Reply:
@timomayer?

Reply:
@SheepFromHeaven we talked about that in person, can you add the hint we talked about, afterwards we can merge it :)

Reply:
@timomayer can you enlighten us, so we know what needs to be done? 👍